src/pages/claim.vue

```diff
@@ -1,9 +1,91 @@
 <script lang="ts" setup>
-import { computed } from 'vue';
-import StatCard from '@/components/cards/StatCard/StatCard.vue';
+import { computed, reactive, watch } from 'vue';
 import { configService } from '@/services/config/config.service';
+import useGaugesQuery from '@/composables/queries/useGaugesQuery';
+import useGaugesDecorationQuery from '@/composables/queries/useGaugesDecorationQuery';
+import { Gauge } from '@/services/balancer/gauges/types';
+import useTokens from '@/composables/useTokens';
+import { PoolToken, PoolType } from '@/services/balancer/subgraph/types';
+import { formatUnits } from 'ethers/lib/utils';
+import useNumbers from '@/composables/useNumbers';
+import { getAddress } from '@ethersproject/address';
+import { isStableLike } from '@/composables/usePool';
+import { useTokenHelpers } from '@/composables/useTokenHelpers';
+import { bnum } from '@/lib/utils';
+import useWeb3 from '@/services/web3/useWeb3';
+import useApp from '@/composables/useApp';
+import useGraphQuery, { subgraphs } from '@/composables/queries/useGraphQuery';
+
+import { RewardRow } from '@/components/tables/BalClaimsTable/BalClaimsTable.vue';
+import BalClaimsTable from '@/components/tables/BalClaimsTable/BalClaimsTable.vue';
 import LegacyClaims from '@/components/contextual/pages/claim/LegacyClaims.vue';
+import GaugeRewardsTable from '@/components/tables/GaugeRewardsTable/GaugeRewardsTable.vue';
+
+/**
+ * TYPES
+ */
+export type GaugePool = {
+  id: string;
+  poolType: PoolType;
+  tokens: PoolToken[];
+  tokensList: string[];
+};
+
+type GaugePoolQueryResponse = {
+  pools: GaugePool[];
+};
+
+type GaugeTable = {
+  gauge: Gauge;
+  pool: GaugePool;
+};
+
+/**
+ * COMPOSABLES
+ */
+const { injectTokens, getToken } = useTokens();
+const { balToken } = useTokenHelpers();
+const { toFiat, fNum2 } = useNumbers();
+const { isWalletReady } = useWeb3();
+const { appLoading } = useApp();
+
+/**
+ * QUERIES
+ */
+const { data: subgraphGauges } = useGaugesQuery();
 
+const gaugesQuery = useGaugesDecorationQuery(subgraphGauges);
+const gauges = computed((): Gauge[] => gaugesQuery.data.value || []);
```

Comment:
I've noticed most of our computed return types are written as `computed<Gauge[]>(() => {})` which I think is easier to read. Are they equivalent?